corehq/toggles/__init__.py

```diff
@@ -707,6 +707,15 @@ def _ensure_valid_randomness(randomness):
               'spaceKey=saas&title=Allow+Configuration+of+Case+List+Tiles',
 )
 
+CASE_LIST_TILE_CUSTOM = StaticToggle(
```

Comment:
Should this have `parent_toggles=[CASE_LIST_TILE]`?

Reply:
Yes. I did not know about that. Learned something new.